lib/matplotlib/pyplot.py

```diff
@@ -1371,30 +1371,23 @@ def ylim(*args, **kwargs):
 def xticks(ticks=None, labels=None, **kwargs):
     """
     Get or set the current tick locations and labels of the x-axis.
-
-    Call signatures::
-
-        locs, labels = xticks()            # Get locations and labels
-        xticks(ticks, [labels], **kwargs)  # Set locations and labels
+    Pass no arguments to return the current values without modifying them.
```

Comment:
Needs an empty line before (Docstring should have a one-line sentence/summary. Everything else should be separate paragraphs).

Reply:
done